css/style.css

```diff
@@ -1,107 +1,182 @@
-     visibility: hidden;
-     display: block;
-     font-size: 0;
-     content: " ";
-     clear: both;
-     height: 0;
-     }
-.clearfix { display: inline-block; }
-/* start commented backslash hack \*/
-* html .clearfix { height: 1%; }
-.clearfix { display: block; }
-/* close commented backslash hack */
-
 .wrapper {
-    margin:  40px 30px;
     border: 3px solid #262626;
-    min-height: 1400px;
     border-radius: 1px;
+    margin:  1.5% 1%;
+    min-height: 1400px;
 }
-.greeting__box {
+.greeting-box {
     position: relative;
-    margin: 100px auto 100px;
+    margin: 8% auto 8%;
     max-width: 400px;
     padding: 5px 10px; 
     border: 2px solid #808080;
-    
+    white-space: nowrap; 
 }
-.greeting__label {
+.greeting-box__label {
+    position: relative;
+    text-transform: uppercase;
     text-align: center;
-    padding-top: auto;
     font-size:40px; 
     font-family: "Trebuchet MS";
-    position: relative;
     font-weight: bold;
     transform: scale;
-    text-transform: uppercase;
-    letter-spacing: 80%;
 }
-.greeting__top-decoration{
+.greeting-box__top-decoration {
     position: absolute;
-    background-color: white;
-    top:-11px;
     left: 50%;
-    font-weight: bold;
+    top:-11px;
     transform: translateX(-50%);
-    text-transform: uppercase;
-    line-height: 20px;
-    text-align: center;
+    display: inline-block;
+    max-width: 300px;
+    overflow: hidden;
     padding: 0 20px;
+    background-color: white;
     color: #808080;
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 20px;
+    text-align: center;
+    text-overflow: ellipsis;
+    text-transform: uppercase;
 }
 .perfomance {
-    max-height: 600px;
-    /*background-color: #1AC0AE;*/
-    position: relative;
-    
-    
-}
-.perfomance_img{
-    width: 20%;
-    float: left;
-    /*background-color: #E4CD0A;*/
-    margin: 5px;
-    margin-left: 20px;
-
-
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+    align-items: flex-start;
+    margin: 0 8% 0 8%;
 }
-.perfomance_table{
-    display: inline-block;
-    margin: 5px;
-    width: 30%;
-    float: left;
+.perfomance__data {
+    display: flex;
+    flex: 1 1 1400px; 
+}
+.perfomance-image {
+    padding: 2%;
+}
+.perfomance__img {
+    width: auto;
+    height: 300px;
+}
+.perfomance__table {
+    margin: 2%;    
+    height: 300px;
+    display: flex;
     border-collapse: collapse;
-    /*background-color: #04D256;  */
-    
+    flex-flow: column wrap; 
+    justify-content: space-between;
 }
-.perfomance__colums{
-    border-bottom: 3px solid #c8c8c8;
+.perfomance-profile {
+    padding-left: 2%;
+    flex-flow: column wrap;
+    flex: 2 1 500px;
+}
+.perfomance-line {
+    display: flex;
+    justify-content: space-between;
+    margin: 1% 0;
+    border-bottom: 2px solid #c8c8c8;
+}
+
+.perfomance-line__row {
     padding: 5px 0;
     font-family: "Trebuchet MS"
 }
-.perfomance__profile {
+.perfomance-profile__title {
```

Comment:
well, you need to update all your classes.

you have
`.perfomance`
`.perfomance-profile`
`.perfomance-line`
`.perfomance_table`

it's made people frustrate. You CANT use element without BLOCK! as you did it below

```
<table class="perfomance_table">
   <tr>
       <td class="perfomance__colums">Name:</td>
```
NEED to update markup and classnaming according to such schema  https://wsd.events/2014/12/06/pres/bem-right/?full#14